internal/scripts/refresh-assets.ts

```diff
@@ -152,7 +152,7 @@ async function copyIcons() {
 }
 
 async function optimizeAndMergeDotcomIcons() {
-	const sourceFolderPath = join(DOTCOM_FOLDER_PATH, 'client', 'public', 'tla')
+	const sourceFolderPath = join(DOTCOM_FOLDER_PATH, 'client', 'assets', 'icons', 'icon')
```

Comment:
Used the same folder structure as we use for our assets package. Might be overkill 🤷‍♂️